apps/deploy-web/src/components/new-deployment/TemplateList.tsx

```diff
@@ -34,15 +33,20 @@ const previewTemplateIds = [
   "akash-network-awesome-akash-grok",
   "akash-network-awesome-akash-FastChat"
 ];
+
 type Props = {
   onChangeGitProvider: (gh: boolean) => void;
+  onTemplateSelected: Dispatch<TemplateCreation | null>;
+  setEditedManifest: Dispatch<SetStateAction<string>>;
```

Comment:
Perhaps this would be a better thing to do?
```suggestion
  onManifestEdited: Dispatch<SetStateAction<string>>;
```
exposing as `Dispatch` kinda breaks the boundary of concerns too
```suggestion
  setEditedManifest: string => void;
```